src/test/lambda/utilities/getMainSourceFile.test.ts

```diff
@@ -317,7 +317,8 @@ describe('getMainSourceFile', async () => {
             await test('python')
             await test('python2.7')
             await test('python3.6')
-       })
+            await test('python3.7')
```

Comment:
Because we're working in a feature branch, could you please make this change in a separate PR aimed at the develop branch? It looks like something we wouldn't want to wait until the dotnet feature branch is complete.

Reply:
Done. Removed `await test('python3.7')`.
✔️